runtime-testsuite/test/org/antlr/v5/test/runtime/RuntimeTestDescriptorParser.java

```diff
@@ -12,8 +12,28 @@
 import java.util.*;
 
 public class RuntimeTestDescriptorParser {
+	private final static String notesSectionName = "notes";
```

Comment:
shouldn't these be spelled as constants i.e. NOTES_SECTION_NAME ?

Reply:
I investigated the probject a bit and came to a conclusion that we use CAPS format for constants, thanks. I've fixed these constant names according to your suggestion in `!fixup` commit (it will be merged with the related commit before the final merge).